plugins/cody-chat/src/com/sourcegraph/cody/chat/ChatView.java

```diff
@@ -153,8 +154,12 @@ private void onStartNewChat(Browser browser) {
     manager.withAgent(
         agent -> {
           try {
-
-            chatId = agent.server.chat_new(null).get(5, TimeUnit.SECONDS);
+            // Resolve the native webview with the pre-defined view IDs.
+            chatId = "eclipse-sidebar";
+            Webview_ResolveWebviewViewParams viewParams = new Webview_ResolveWebviewViewParams();
+            viewParams.viewId = "cody.chat";
```

Comment:
Is this a magic value or just something convenient?

Reply:
chatID is just convenient, but "cody.chat" is what we use to register sidebar view in VS Code for the when clause. I supposed it shouldnt have any effect on agent (not very sure though)?